### PR TITLE
SEO-2520-Retarget-FF-docs-PDF

### PR DESCRIPTION
**Pull request by Gowri-V-L**
Hi All,

Worked for retargeting keywords with the word C#/.NET in PDF UG.
pdf annotation .net
pdf annotation c#
compress pdf .net
compress pdf c#
pdf digital signature .net
pdf digital signature c#

### CODE REVIEW

**Comment by Gowri-V-L** [comment]
rebuild

**Comment by Gowri-V-L** [comment]
rebuild

File: File-Formats/PDF/Working-with-Compression.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ documentation: UG
 ---
 # Working with Compression
 
-Essential PDF allows you to [compress the PDF](https://www.syncfusion.com/document-processing/pdf-framework/net/pdf-library/compress-pdf) document and thereby reduce the file size in the following three ways.
+Essential PDF allows you to [compress .NET PDF](https://www.syncfusion.com/document-processing/pdf-framework/net/pdf-library/compress-pdf) document and thereby reduce the file size in the following three ways.
```
[the remark this answers is not in the document]
Comment (Gowri-V-L) [reply File-Formats/PDF/Working-with-Compression.md]: removed

File: File-Formats/PDF/Working-with-Annotations.md [hunk 2]
```diff
@@ -12269,4 +12269,9 @@ freeText.SetAppearance(true);
...
 </tr>
 </table>
-</table>
\ No newline at end of file
+</table>
+
+## See Also
+[How to modify the size of C# PDF Annotation](https://www.syncfusion.com/kb/10290/how-to-modify-the-size-of-the-annotation-in-pdf-using-c-and-vb-net)
+
+N> You can also explore our [C# PDF Annotation](https://www.syncfusion.com/document-processing/pdf-framework/net/pdf-library/pdf-annotation) feature tour page that how to use .NET PDF with 20+ annotation types with built-in support, or create their own custom annotations and [PDF Annotaions .NET demo](https://ej2.syncfusion.com/aspnetmvc/PDF/AnnotationFlatten#/material) that demonstrates the creation of different types of annotations such as ink, free text, pop up, text markup annotation, and more.
```
Comment (Girija-Rajendran) [inline File-Formats/PDF/Working-with-Annotations.md:12277]: Get the content review for this
image

**Comment by Gowri-V-L** [comment]
rebuild

File: File-Formats/PDF/Working-with-Annotations.md (hunk 2 above)
Comment (Gowri-V-L) [reply File-Formats/PDF/Working-with-Annotations.md]: got it

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**